### ISSUE TITLE
Improve JSON Error message format

### ISSUE THREAD

**Issue by hdoupe** [PSLmodels/ParamTools#61]
The JSON error messages need to be improved. ParamTools uses a utility function in `utils.py` named `read_json` to read the defaults and adjustment specs:

```python
def read_json(path):
    """
    Read JSON file shortcut
    """
    if isinstance(path, str) and os.path.exists(path):
        with open(path, "r") as f:
            r = json.loads(f.read(), object_pairs_hook=OrderedDict)
        return r
    elif isinstance(path, dict):
        return path
    else:
        return json.loads(path)
```

`read_json` checks if the input is a file path, then it checks if it is a Python dictionary, and finally, assumes it is a string containing JSON formatted data. If the JSON can not be read, then ParamTools should print the input, indicate whether it is a valid path, and indicate whether the input is valid JSON. This would give the user more information about why the read failed and would give them a better idea for how to fix it. ParamTools could look to Tax-Calculator to see how it reads JSON formatted data.

**Comment by hdoupe** [comment]
See https://github.com/PSLmodels/Cost-of-Capital-Calculator/pull/269#discussion_r296030607 for this issue's motivation.

**Comment by hdoupe** [comment]
Resolved #61 via #111.
